mmdet3d/models/detectors/centerpoint.py

```diff
@@ -79,3 +81,86 @@ def simple_test_pts(self, x, img_metas, rescale=False):
             for bboxes, scores, labels in bbox_list
         ]
         return bbox_results
+
+    def aug_test_pts(self, feats, img_metas, rescale=False):
+        """Test function of point cloud branch with augmentaiton."""
```

Comment:
Add the steps of the function in the docstring thus the users could better understand the implementation of the function. For example
```python
Step 1: map all the feature map back
Step 2: xxxxx
Step 3: xxxxx
```